### PR TITLE
Added a Sheep class 

### PR DESCRIPTION
**Pull request by Enidx**
That implements the interface Tradable and Domesticable. Also created a test file for this class.

### CODE REVIEW

File: src/main/java/Sheep.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class Sheep implements Domesticatable, Tradable{
+    /* A Sheep class that implements the interface Domesticable and Drivable
```
Comment (DavidM0808) [inline src/main/java/Sheep.java]: Maybe specify a constructor after this comment here.

File: src/main/java/Trader.java [hunk 2]
```diff
@@ -96,8 +96,8 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
```
Comment (DavidM0808) [inline src/main/java/Trader.java]: Need to add a comma here.

File: src/test/java/SheepTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import static org.junit.Assert.assertEquals;
+
+public class SheepTest {
+    Sheep s;
+
+    @Before
+    public void setUp() throws Exception {
+        s = new Sheep();
+    }
+
```
Comment (DavidM0808) [inline src/test/java/SheepTest.java:13]: You can add a general documentation of the test cases here.

**Review by DavidM0808 — CHANGES_REQUESTED** [review]

File: src/main/java/Sheep.java (hunk 1 above)
Comment (Winnie1029) [reply src/main/java/Sheep.java]: The class has implemented Tradable not Drivable. The documentation need to be changed.

**Review by Winnie1029 — CHANGES_REQUESTED** [review]

**Comment by Enidx** [comment]
Finished!

**Pull request merged**